xpanse/api/incident_management/v1/incidents.py

```diff
@@ -0,0 +1,182 @@
+        return XpanseResponse(response, data_key=self.DATA_KEY)
+
+    def count(
+        self, request_data: Optional[RequestData] = None, **kwargs: Any
+    ) -> XpanseResponse:
+        """
+        This endpoint will return a count of Incidents.
+
+        Args:
+            request_data (RequestData, Optional):
+                Any supplemental request_data to be included with your request. This is needed to
+                implement any additional filters, offsets, limits, or sort ordering.
+            **kwargs:
+                Any extraneous parameters you would like to include when executing your
+                request with the Requests.request module. Note: By default, all payload data
+                is sent under the "json" keyword for your request.
+
+        Returns:
+            :obj:`XpanseResponse`:
+                An object containing the raw requests.Response and parsed data results.
+                The raw response can be accessed with `<xpanse_reponse>.response` attribute.
+                The parsed results can be accessed with the `<xpanse_response>.data` attribute.
+
+        Examples:
+            >>> # Get Incidents total count:
+            >>> incidents =  client.incidents.count()
+            >>> if incidents.response.status_code < 300:
+            >>>     count = incidents.data
+        """
+        return super(IncidentsEndpoint, self)._count(
+            self.LIST_ENDPOINT, request_data=request_data, **kwargs
+        )
+
+    def update(
+        self, incident_ids: List[str], update_data: Any, **kwargs: Any
+    ) -> XpanseResponse:
+        """
+        This endpoint will update a set of Incidents' data.
+
+        Args:
+            incident_ids (List[str]):
+                The list of Incident ids to modify with your request data.
+            update_data (Any):
+                The data with which to update the set of Incidents.
+            **kwargs:
+                Any extraneous parameters you would like to include when executing your
+                request with the Requests.request module. Note: By default, all payload data
+                is sent under the "json" keyword for your request.
+
+        Returns:
+            :obj:`XpanseResponse`:
+                An object containing the raw requests.Response and parsed data results.
+                The raw response can be accessed with `<xpanse_reponse>.response` attribute.
+                The parsed results can be accessed with the `<xpanse_response>.data` attribute.
+
+        Examples:
+            >>> # Update Incidents with new assignee:
+            >>> incidents =  client.incidents.update(incident_ids=["id1", "id2"],
+            >>>                                      update_data={"assigned_user_mail": "new@mail.com"})
+            >>> if incidents.response.status_code < 300:
+            >>>     results = incidents.data
+
+        """
+        extra_request_data = {
+            "incident_id_list": incident_ids,
```

Comment:
I think this should just be `incident_id`, update can only apply to a single incident as far as I can tell.

Reply:
good catch - you're right (https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-API-Reference/Update-an-Incident) 

the example here is wrong, but the table listing the request fields is right - I'll confirm in the api docs with Charissa that we have the right set up